doc/level0.rst

```diff
@@ -57,16 +57,16 @@ decrypting.  It MUST be capable of assembling these keys into an
 OpenPGP certificate (RFC 4880 "Transferable Public Key") that
 indicates these capabilities.
 
-These secret keys MUST be protected from access by other applications
-or co-tenants of the device at least as well as the passwords the MUA
-retains for the user's IMAP or SMTP accounts.  These secret keys MUST
-never be sent over the wire to any other party.
+The secret key material is critical for security as in other end-to-end
+applications, and should be protected from access by other applications or
+co-tenants of the device, at least as well as the passwords the MUA retains for
+the user's IMAP or SMTP accounts.
 
-When an Autocrypt-enabled MUA configures an e-mail account, it should
-generate these keys and store them locally.  Then it should proceed to 
-try to "claim" the account to `lock out <lockout>`_ other MUAs of the
-same users.  In Level 0 only one MUA can send and receive encrypted
-mail through Autocrypt mechanisms.
+In Level 0 only one MUA can send and receive encrypted mail through Autocrypt
+mechanisms.  When an Autocrypt-enabled MUA configures an e-mail account, it
+first tries to "claim" the account, to `lock out <lockout>`_ other MUAs of the
+same users.  If the claim was successful, the MUA should proceed to generate
```

Comment:
I actually think a responsible MUA should generate provisional keys *before* attempting to claim the account, to minimize the race window -- that way, if the account appears to be unclaimed, it can claim it immediately.